### DISCUSSION TITLE
Process for updating remote kenv?

### DISCUSSION THREAD

**Discussion by dealingwith** [johnlindquist/kit#348] [Q&A]
I added a remote kenv using `Clone repo of scripts`:

<img width="356" alt="Screen Shot 2021-07-14 at 3 31 18 PM" src="https://user-images.githubusercontent.com/124208/125688948-77dbd53e-a007-4ac3-b6d0-759a7f086da8.png">

The remote repo was then updated. In order to get Kit to update, I had to:

1. manually git pull from the `.kenv` subfolder containing that repo
2. restart Kit

Is that the correct workflow at present, or did I miss some magic? ✨

**Comment by johnlindquist** [comment]
You didn't miss anything. I'll add an "Update Kenv" in the "Kit" section to do a `git pull` behind the scenes. Otherwise, yes, it's just manually managing a git repo.
